samples/MvcSandbox/project.json

```diff
@@ -17,14 +17,24 @@
   },
   "content": [
     "Views",
-    "web.config"
+    "web.config",
+    "wwwroot"
   ],
+  "tools": {
+    "dotnet-razor-tooling": {
+      "version": "1.0.0-*",
+      "imports": [
+        "net45"
```

Comment:
Does this bit actually work? Can verify by looking at the lock file and verifying that it says that dotnet-razor-tooling is a lock file and not `{}`